installer/arm64/install.sh

```diff
@@ -99,6 +99,12 @@ if [ "$install_env" = "onie" ]; then
 elif [ "$install_env" = "sonic" ]; then
     demo_mnt="/host"
     eval running_sonic_revision=$(cat /etc/sonic/sonic_version.yml | grep build_version | cut -f2 -d" ")
```

Comment:
Still, relying on /etc/sonic/sonic version.yml is not robust enough. Probably, we need to improve a logic which retrieves the current running image. Maybe /proc/cmdline will be good enough...

Reply:
`/proc/cmdline` should be much better solution. Even you checked the image exists on disk, it may not the running version.

---
In reply to: [610042919](https://github.com/Azure/sonic-buildimage/pull/7264#discussion_r610042919) [](ancestors = 610042919)

Reply:
@qiluo-msft fixed